org.gnome.Contacts.json

```diff
@@ -0,0 +1,215 @@
+{
+    "app-id": "org.gnome.Contacts",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-contacts",
+    "tags": [ "stable" ],
+    "rename-icon": "gnome-contacts",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.ControlCenter",
+        "--talk-name=org.gnome.OnlineAccounts",
+        "--talk-name=org.gnome.evolution.dataserver.AddressBook9",
+        "--talk-name=org.gnome.evolution.dataserver.Sources5",
+        "--talk-name=org.gnome.evolution.dataserver.Subprocess.Backend.*",
+        "--talk-name=org.freedesktop.Telepathy.AccountManager",
+        "--filesystem=home:ro",
```

Comment:
What does it need home access for?

Reply:
To allow selecting an avatar image from a file.

Reply:
If you switch https://github.com/GNOME/gnome-contacts/blob/b4822d1040af1ff5858fe22924acbfb75a96ca7d/src/contacts-avatar-selector.vala#L225 for [FileChooserNative](https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html) then you can drop this as it'll use the portal. I've filed this upstream as https://gitlab.gnome.org/GNOME/gnome-contacts/issues/96 it won't block this PR, but it'd be great to see it fixed in a future release.